components/Player.js

```diff
@@ -1,32 +1,58 @@
-import React from 'react';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import Head from 'next/head';
-import Link from 'next/link';
-
-import { createShowDate, durationToHHMMSS, removeLeadingZero, splitShowDate } from '../lib/utils';
-import player from '../lib/player';
+import React from "react";
```

Comment:
hmm looks like your personal eslint/prettier config overrode the one in the project, mind using ours?